graphql/util.go

```diff
@@ -380,22 +380,6 @@ func modifyVersionHandler(ctx context.Context, patchID string, modification mode
 	if err != nil {
 		return mapHTTPStatusToGqlError(ctx, httpStatus, err)
 	}
-
```

Comment:
Curious why was this removed? My understanding was that we'd add logic elsewhere to match what is happening here, not remove the child patch logic from the graphQL helper.